README.rst

```diff
@@ -0,0 +1,77 @@
+===========
+itemloaders
+===========
+
+.. image:: https://img.shields.io/pypi/v/itemloaders.svg
+   :target: https://pypi.python.org/pypi/itemloaders
+   :alt: PyPI Version
+
+.. image:: https://img.shields.io/pypi/pyversions/itemloaders.svg
+   :target: https://pypi.python.org/pypi/itemloaders
+   :alt: Supported Python Versions
+
+.. image:: https://travis-ci.com/scrapy/itemloaders.svg?branch=master
+   :target: https://travis-ci.com/scrapy/itemloaders
+   :alt: Build Status
+
+.. image:: https://codecov.io/github/scrapy/itemloaders/coverage.svg?branch=master
+   :target: https://codecov.io/gh/scrapy/itemloaders
+   :alt: Coverage report
+
+.. image:: https://readthedocs.org/projects/itemloaders/badge/?version=latest
+   :target: https://itemloaders.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
+
+
+``itemloaders`` is a library that helps you collect data from XML and HTML sources into models.
```

Comment:
```suggestion
``itemloaders`` is a library that helps you collect data from HTML and XML sources.
```

Changed the order or HTML and XML, and removed "models" - it was not clear for me what "models" mean.